parser_library/src/expressions/conditional_assembly/terms/ca_expr_list.cpp

```diff
@@ -204,7 +206,7 @@ struct resolve_stacks
     {
         while (!op_stack.empty())
         {
-            if (op_stack.top().priority + right_assoc > prio_limit)
+            if (op_stack.top().priority + right_assoc >= prio_limit)
```

Comment:
doesn't this break left-associative operators (turn them essentially into right-associative)?